gene_benchmark/tasks.py

```diff
@@ -413,6 +416,8 @@ def load_task_definition(
         frac (float): load a unique fraction of the rows in the task, default 1
         tasks_folder(str|None): Use an alternative task repository (default repository if None)
         sub_task(str|None): Use only one of the columns of the outcome as a binary task
+        multi_label_th (float): Threshold for multi label tasks outcomes
+        cat_label_th (float): Threshold for categorical tasks outcomes
```

Comment:
I would add, only categories that have rates above the threshold will be included.